backup_and_restore/index.adoc

```diff
@@ -33,45 +33,44 @@ You can always recover from a disaster situation by xref:../backup_and_restore/c
-//
-// .OADP requirements
-//
-// OADP has the following requirements:
-//
-// * You must be logged in as a user with a `cluster-admin` role.
-// * You must have object storage for storing backups, such as one of the following storage types:
-//
-// ** Amazon Web Services
-// ** Microsoft Azure
-// ** Google Cloud Platform
-// ** Multicloud Object Gateway
-// ** S3-compatible object storage, such as Noobaa or Minio
-//
-// :FeatureName: The `CloudStorage` API for S3 storage
-// include::modules/technology-preview.adoc[]
-//
-// * To back up PVs with snapshots, you must have cloud storage that has a native snapshot API or supports Container Storage Interface (CSI) snapshots, such as the following providers:
-//
-// ** Amazon Web Services
-// ** Microsoft Azure
-// ** Google Cloud Platform
-// ** CSI snapshot-enabled cloud storage, such as Ceph RBD or Ceph FS
-//
-// [NOTE]
-// ====
-// If you do not want to back up PVs by using snapshots, you can use link:https://restic.net/[Restic], which is installed by the OADP Operator by default.
-// ====
-//
-// .Backing up and restoring applications
-//
-// You back up applications by creating a xref:  ../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications[`Backup`]  custom resource (CR). You can configure the following backup options:
-//
-// * xref:  ../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-hooks_backing-up-applications[Backup hooks] to run commands before or after the backup operation
-// * xref:  ../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-scheduling-backups_backing-up-applications[Scheduled backups]
-// * xref:  ../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-applications-restic_backing-up-applications[Restic backups]
-//
-// You restore applications by creating a xref:  ../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[`Restore`] CR. You can configure xref:  ../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-hooks_restoring-applications[restore hooks] to run commands in init containers or in the application container during the restore operation.
+OADP backs up and restores Kubernetes resources and internal images, at the granularity of a namespace, by using link:https://velero.io/[Velero 1.7]. OADP backs up and restores persistent volumes (PVs) by using snapshots or Restic. For details, see xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-features_oadp-features-plugins[OADP features].
+
+.OADP requirements
```

Comment:
This and `Backing up and restoring applications` below should probably be `==` headings, with IDs and `[discrete]` if you do not want them to appear in the TOC. Among other things, this allows people to link to the section if needed. 

I don't see this explicitly prohibited in the guidelines, but based on talking to another reviewer we're pretty sure the `.Heading` format should be reserved for the types of examples given in the guidelines (e.g. `.Prerequisites`).